examples/src/examples/graphics/index.mjs

```diff
@@ -69,7 +69,7 @@ export {
     ClusteredOmniShadowsExample,
     ClusteredSpotShadowsExample,
     ContactHardeningShadowsExample,
-    // LitMaterialExample,
+    LitMaterialExample,
```

Comment:
and here please remove this line, and also the relevant import in the same file - the example is not ready to be shipped yet.

Reply:
No worries.

Reply:
Remove or comment ?  So you can quickly undo this ?